lib/metrics.coffee

```diff
@@ -48,6 +49,15 @@ module.exports =
       return if eventName of IgnoredCommands
       Reporter.sendCommand(eventName)
 
+    grim.on 'updated', (deprecation) ->
+      setImmediate ->
+        for path, stack of deprecation.stacks
+          packageName = stack.metadata?.packageName
+          pack = atom.packages.getLoadedPackage(packageName)
+          version = pack?.metadata?.version
+          Reporter.sendEvent('deprecation', packageName, version) if packageName?
```

Comment:
Ah, I was optimistic that it was happening at a higher level for all deprecations. Thanks